website/content/partials/auto-roll-billing-start-example.mdx

```diff
@@ -0,0 +1,7 @@
+The default billing start date used in queries and license utilization reporting will be automatically adjusted to the latest billing year. Past year billing information will still be retained.
+
+For example, Assume a customer has signed a deal for 3 years with a license start date of Dec 1 2023. After upgrading to the version with this change, the billing start date 
+auto-rolls to the most recent year.
+- If the upgrade happens on Sept 1 2024, the billing start date will continue to be Dec 1 2023. When we are on Dec 1 2024, the billing start date 
+will automatically roll to Dec 1, 2024. 
+- If the upgrade happens on Dec 15 2024, the billing start date should automatically roll to Dec 1, 2024 after upgrade.
```

Comment:
@schavis I feel like you probably have some good recommendations for the layout and wording here, so I will defer to you.